src/handlers/options.ts

```diff
@@ -12,9 +12,20 @@ export function getOptionHandler(dbPool: PostgresJsDatabase<typeof db>) {
       return res.status(400).json({ error: 'Missing optionId' });
     }
 
-    const option = await dbPool.query.questionOptions.findFirst({
-      where: eq(db.questionOptions.id, optionId),
-    });
+    const option = await dbPool
+      .select({
+        Id: db.questionOptions.id,
```

Comment:
does this return Id with a capital letter in the object? If so, convert this to `id`

Reply:
good catch